core/src/main/java/de/bwaldvogel/mongo/wire/MongoDatabaseHandler.java

```diff
@@ -78,6 +81,11 @@ protected void channelRead0(ChannelHandlerContext ctx, ClientRequest object) thr
         } else if (object instanceof MongoUpdate) {
             MongoUpdate update = (MongoUpdate) object;
             mongoBackend.handleUpdate(update);
+        } else if (object instanceof MongoGetMore) {
```

Comment:
What happens when cursor is [closed](https://mongodb.github.io/mongo-java-driver/3.6/javadoc/com/mongodb/Cursor.html#close--) ?

Reply:
Good point. I added support for this on commit eb83213
The response from the server will include a flag 0 indicating the cursor does not exists
https://docs.mongodb.com/manual/reference/mongodb-wire-protocol/#wire-op-reply